### ISSUE TITLE
Accessibility - Step 7 Topic 1 - Screen Needs Heading Level Three (h3) Items

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#59]
The bold text under the "Example In Action" heading level two (h2) should be heading level three (h3).

The same will apply to any other screens that have a similar visual structure.

**Comment by adriennerlong** [comment]
John, can we make the full title "Example in Action: The Fair Labor Association's (FLA) Accreditation Program for Independent External Monitors" a heading level two (h2)?

**Comment by jfc3-dol** [comment]
@adriennerlong,

If you get rid of the "Example in Action" as the heading level two (h2). If not then that text should be a heading level three (h3).

**Comment by adriennerlong** [comment]
ICF, we would like to keep "example in action" as a header. So please make the specific example name a higher level [e.g. if Example in Action if (h3) then make Better Work Remediation Program (h4)].

**Comment by jfc3-dol** [comment]
The same applies to all the other Topic pages and the Step pages too for heading structure. You shouldn't skip a level.

**Comment by jfc3-dol** [comment]
@adriennerlong,

It looks to me that the "Example in Action" text and the text following it have been combined to make one line which is a heading level two (h2).

Going to close the issue and assume the changes have been made to other screens too.
